book-examples/leptos/Cargo.toml

```diff
@@ -33,8 +36,11 @@ button = [
     "radix-leptos-icons/envelope-open",
     "radix-leptos-icons/reload",
 ]
+alert = [
+    "dep:shadcn-ui-leptos-alert"
+]
```

Comment:
Please keep this sorted alphabetically.

When using icons in examples, the optional dependency needs to be enabled here, see the Yew book examples.